easybuild/easyconfigs/c/CPPE/CPPE-0.3.1-GCC-12.2.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'CMakeMake'
+
+name = 'CPPE'
+version = '0.3.1'
```

Comment:
@laraPPr There's a `0.3.2` tag, let's use that instead?

Reply:
yeah I wasn't sure that worked since it does not seem to be an official release

Reply:
OK, let's stick to 0.3.1 then, but open an issue to ask why 0.3.2 is not on PyPI (and not tagged as release)

Reply:
https://github.com/maxscheurer/cppe/issues/45